src/client/CourseRequestForm/index.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+// before mounting your React application:
+import { theme } from '@instructure/canvas-theme'
+theme.use();
+
+import { Button } from '@instructure/ui-buttons'
+import { Checkbox, CheckboxGroup } from '@instructure/ui-checkbox'
+import { Flex } from '@instructure/ui-flex'
+import { FormFieldGroup } from '@instructure/ui-form-field'
+import { Table } from '@instructure/ui-table';
+import { TextInput } from '@instructure/ui-text-input'
+
+const CourseRequestForm = ({courses}) => {
+    const [emailToggleValue, toggleEmail] = useState(false);
+    const handleEmailToggle = (() => {
+        toggleEmail(!emailToggleValue);
+    });
+    const CrossListings = ({term, courseInfoList}) => {
+        if (courseInfoList.length !== 0) {
+            return (
+                courseInfoList.map((courseInfo, index) => (
+                    <Checkbox
+                        key={index}
+                        size='small'
+                        label={ term + '-' + courseInfo.classID + ' (' + courseInfo.subjectAreaCode + ' ' + courseInfo.courseCatalogNumberDisplay + '-' + courseInfo.classNumber + ')' }
+                        defaultChecked
+                    >
+                    </Checkbox>
+                ))
+            );
+        }
+        return null;
+    }
+    const courseListings = courses.map((course, index) => (
+        <Table.Row key={index}>
+            <Table.Cell>{course.requestID}</Table.Cell>
+            <Table.Cell>{course.offeredTermCode}</Table.Cell>
+            <Table.Cell>{course.classID}</Table.Cell>
+            <Table.Cell>{course.subjectAreaCode}</Table.Cell>
+            <Table.Cell>{ course.courseCatalogNumberDisplay + '-' + course.classNumber }</Table.Cell>
+            <Table.Cell>
+                <CrossListings term={course.offeredTermCode} courseInfoList={course.crosslistedCourses}></CrossListings>
+                <TextInput width='200px' renderLabel=''/>
+                <Button>Add additional Class ID</Button>
+            </Table.Cell>
+            <Table.Cell>{course.timeRequested}</Table.Cell>
+            <Table.Cell>
+                <TextInput width='200px' type='email' renderLabel=''/>
+            </Table.Cell>
+            <Table.Cell>{course.status}</Table.Cell>
+            <Table.Cell>
+                <Flex justifyItems='center'>
+                    <Flex.Item>
+                        <Checkbox label='' size='small'/>
+                    </Flex.Item>
+                </Flex>
+            </Table.Cell>
+            <Table.Cell>
+                <Flex justifyItems='center'>
+                    <Flex.Item>
+                        <Checkbox label='' size='small' defaultChecked/>
```

Comment:
@kubilayagi The default check value for "Do NOT send URL to MyUCLA" should come from a config value. So for test environments we want it to be off. For production, we want it to default to true.

Reply:
@rlorenzo removed. Should I leave the defaultChecked for "To be built" though?

Reply:
To be built should be checked by default